ir/pass_manager.h

```diff
@@ -36,6 +36,7 @@ class PassManager : virtual public Visitor, virtual public Backtrack {
     void addPasses(const std::initializer_list<Visitor *> &init) {
         never_backtracks_cache = -1;
         for (auto p : init) if (p) passes.emplace_back(p); }
+    void deletePasses() { passes = vector<Visitor *>(); }
```

Comment:
Not sure why this would be needed -- generally passes are only added when the PassManager is created and never change.  The only reason `addPasses` exists (rather than just using the initializer_list constructor) is historical accident.

Reply:
Please see reply to cc10512 above.